pkg/jetcloud/client.go

```diff
@@ -26,12 +25,13 @@ type errorResponse struct {
 	} `json:"error"`
 }
 
-func newClient() *client {
+func (jc *JetCloud) newClient() *client {
```

Comment:
This is a bit fragile. Maybe we can merge the api object and the client into one?

Reply:
yeah, lets merge now that we have another struct